src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -87,7 +140,39 @@ def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_w
             location=location,
             tags=tags)
 
-    return client.create(resource_group_name, server_name, parameters)
+    server_result = resolve_poller(
+        client.create(resource_group_name, server_name, parameters), cmd.cli_ctx,
+        '{} Server Create'.format(engine_name))
+    user = server_result.administrator_login
+    server_id = server_result.id
+    loc = server_result.location
+    version = server_result.version
+    sku = server_result.sku.name
+    host = server_result.fully_qualified_domain_name
+
+    # Adding firewall rule
+    if public_network_access is not None and start_ip != '':
+        firewall_id = create_firewall_rule(cmd, resource_group_name, server_name, start_ip, end_ip, engine_name)
+
+    logger.warning('Make a note of your password. If you forget, you would have to '
+                   'reset your password with \'az %s flexible-server update -n %s -g %s -p <new-password>\'.',
+                   engine_name, server_name, resource_group_name)
+
+    update_local_contexts(cmd, server_name, resource_group_name, location, user)
+
+    if engine_name == 'postgres':
+        return form_response(user, sku, loc, server_id, host, version,
+                             administrator_login_password if administrator_login_password is not None else '*****',
+                             create_postgresql_connection_string(server_name, host, user, administrator_login_password),
+                             infrastructure_encryption, None, firewall_id)
+    # Serves both - MySQL and MariaDB
+    # Create mysql database if it does not exist
+    database_name = DEFAULT_DB_NAME
+    create_database(cmd, resource_group_name, server_name, database_name, engine_name)
```

Comment:
For previous design, _server_create() is only used to create server, but here you add a following operation to create database, right?
Will it break some existing commands which use _server_create()?

Reply:
Should not break anything as the new _server_create() **creates a server + creates a database.**.